### PR TITLE
fix(docs-infra): do not include GitHub links in Table of Content

### PR DESCRIPTION
**Pull request by gkalpak**
The docs template for cli commands ([cli-command.template.html][1]) includes an `h2` element with GitHub links for [long description][2]. Since the content of `h2` elements is replicated in the auto-generated Table of Contents, the GitHub links were replicated as well (which is undesirable).

This commit fixes it by explicitly excluding `.github-links` elements, when extracting the content for the ToC (in [TocService#extractHeadingSafeHtml()][3]). This is similar to what we do for the auto-generated `.header-link` anchors.

[1]: https://github.com/angular/angular/blob/1537791f0/aio/tools/transforms/templates/cli/cli-command.template.html
[2]: https://github.com/angular/angular/blob/1537791f0/aio/tools/transforms/templates/cli/cli-command.template.html#L18
[3]: https://github.com/angular/angular/blob/1537791f0/aio/src/app/shared/toc.service.ts#L56

### CODE REVIEW

File: scripts/git/commit-msg.js [hunk 1]
```diff
@@ -24,7 +24,10 @@ if (msgFile) {
     isValid = checkMsg(firstLine);
 
     if (!isValid) {
-        console.error('\nCheck CONTRIBUTING.md at the root of the repo for more information.');
+        console.error(
+            '\nCheck CONTRIBUTING.md at the root of the repo for more information.' +
+            '\n' +
+            '\n(In case you need the invalid commit message, it should be stored in \'.git/COMMIT_EDITMSG\'.)');
```
Comment (petebacondarwin) [inline scripts/git/commit-msg.js]: LOL did you get bitten by this?

**Review by petebacondarwin — APPROVED** [review]

**Review by petebacondarwin — CHANGES_REQUESTED** [review]
Actually there seem to be a bunch of legitimate failures.

File: scripts/git/commit-msg.js (hunk 1 above)
Comment (gkalpak) [reply scripts/git/commit-msg.js]: More times than I want to admit 😇

(I thought I had removed this change from this PR. I submitted #32420.)

**Comment by gkalpak** [comment]
It turns out there was another issue with tooltips (`title` attribute) of ToC items that the tests didn't hit before. I added another commit to fix tha too.

@petebacondarwin: PTAL 👀

**Review by petebacondarwin — APPROVED** [review]

**Pull request closed without merging**